README.md

```diff
@@ -14,15 +14,16 @@ urlFragment: azure-iot-samples-for-csharp-net
 
 azure-iot-samples-csharp provides a set of easy-to-understand, continuously-tested samples for connecting to Azure IoT Hub via Azure/azure-iot-sdk-csharp.
 
-## Prerequisites
+These samples are against the `preview` releases of the Azure IoT Hub SDK for .NET. For samples against the GA releases, see [here](https://github.com/Azure-Samples/azure-iot-samples-csharp).
 
-- .NET Core SDK 3.0.0 or greater on your development machine.  You can download the .NET Core SDK for multiple platforms from [.NET](https://www.microsoft.com/net/download/all).  You can verify the current version of C# on your development machine using 'dotnet --version'.
+> NOTE: The `preview` branch is meant to hold only those samples that are relevant to the corresponding `preview` feature added in the SDK. Once the `preview` feature is GA'd, these samples will be moved over to the main branch. This branch is not intended to be directly merged into the main branch.
```

Comment:
@drwill-ms I added this note calling out our branching strategy. Let me know if you have any suggestions.